### ISSUE TITLE
Undefined is not an object (React.PropTypes.func)

### ISSUE THREAD

**Issue by wayne1203** [xiewang/react-native-emoticons#15]
I'm using RN 0.49 and emoticons 1.0.7
_onEmoticonPress and _onBackspacePress are implemented 
and showEmoticons has been defined in constructor
```
<Emoticons
   onEmoticonPress={this._onEmoticonPress.bind(this)}
   onBackspacePress={this._onBackspacePress.bind(this)}
   show={this.state.showEmoticons}
   concise={true}
   showHistoryBar={true}
   showPlusBar={true}
/>
```

![screenshot_1509078165](https://user-images.githubusercontent.com/8578341/32087857-8b6b4a9a-bb11-11e7-87b1-abf9c72ae6ab.png)

**Comment by xiewang** [comment]
refer to #14

**Comment by wayne1203** [comment]
Thank you.
After I changed all the files following the pull request and edited "DefaultTabBar.js" for using 'prop-types'
I got another error... It's the same problem (Due to PropTypes)
Seems like are the problems of react-native-scrollable-tab-view

**Comment by wayne1203** [comment]
Fixed by downgraded to RN 0.48.1
